tools/c7n_azure/c7n_azure/function_package.py

```diff
@@ -99,11 +99,14 @@ def _add_function_config(self):
             binding['schedule'] = self.policy['mode']['schedule']
 
         elif mode_type == 'azure-stream':
```

Comment:
Do we want to keep this name? I guess it doesn't matter what the underlying trigger is if the semantics of the mode makes sense.

Reply:
probably not - so much of this is going to get a rewrite I'm mostly just trying to unblock and we can finalize naming once we see what it does.